bin/main

```diff
@@ -1,53 +1,85 @@
 #!/usr/bin/env ruby
-def display_board(board)
-    puts " #{board[0]} | #{board[1]} | #{board[2]} "
-    puts "-----------"
-    puts " #{board[3]} | #{board[4]} | #{board[5]} "
-    puts "-----------"
-    puts " #{board[6]} | #{board[7]} | #{board[8]} "
-end
+require_relative '../lib/game_logic'
+require_relative '../lib/players'
+
+puts 'Welcome to Tic-Tac-Toe'
+sleep(1)
+puts 'Before we start may we know your names'
+sleep(1)
 
 game_on = true
+
 until game_on == false
```

Comment:
**OPTIONAL** As indicated by the previous reviewer, is better to write `!game_one`

Reply:
Solved!